code/game/supplyshuttle.dm

```diff
@@ -168,21 +168,14 @@ var/list/mechtoys = list(
 /datum/controller/supply/New()
 	ordernum = rand(1,9000)
 
-//Supply shuttle ticker - handles supply point regeneration and shuttle travelling between centcomm and the station
-/datum/controller/supply/proc/process()
 	for(var/typepath in subtypesof(/datum/supply_packs))
 		var/datum/supply_packs/P = new typepath()
-		if(P.name == "HEADER")
-			qdel(P)
-			continue
+		if(P.name == "HEADER") continue

```

Comment:
Assuming "HEADER" is the initial value for the name field one can avoid new() for such entries by doing:
```
for(var/typepath in subtypesof(/datum/supply_packs))
  var/datum/supply_packs/P = typepath
  if(initial(P.name) == "HEADER")
    continue
  P = new P()
  supply_packs[P.name] = P
```